public/docs/_examples/toh-5/dart/lib/heroes_component.dart

```diff
@@ -26,7 +26,11 @@ class HeroesComponent implements OnInit {
   Hero selectedHero;
 
   // #docregion renaming
-  HeroesComponent(this._heroService, this._router);
+  HeroesComponent(this._heroService,
```

Comment:
Btw, I was trying to avoid having a `heroes_component_1.dart` file (which would be another way of solving this).